### ISSUE TITLE
handleTouch: false bug on tap.

### ISSUE THREAD

**Issue by billythekid** [imgix/drift#273]
**Describe the bug**
When passing `false` as the `handleTouch` param on Drift objects tapping the image brings up the zoom pane and this can't be dismissed. Tapping around the image re-sets the zoom focalpoint much as dragging would were handleTouch enabled. It can't be dragged away or 'touchend-ed'

**To Reproduce**
[Forked Codepen showing issue](https://codepen.io/billythekid/pen/WqZwmW)

Steps to reproduce the behaviour:

1.  Go to https://codepen.io/billythekid/pen/WqZwmW on a touchscreen device
2.  Tap (don't hold) on the watch image
3.  Observe hover pane.

**Expected behaviour**
Nothing should happen on tap, the whole plugin should be 'disabled' on touchscreen when handleTouch is false I believe?

**Information:**

- drift version: 1.3.4 (from npm)
- browser version: iPhone Chrome/Safari/Firefox

**Comment by sherwinski** [comment]
Hi @billythekid, I agree that this is unexpected behavior especially since it was addressed in #220 and released as part of [v1.3.4](https://github.com/imgix/drift/releases/tag/v1.3.4). I compared the version of [Drift.min.js](https://s3-us-west-2.amazonaws.com/s.cdpn.io/164071/Drift.min.js) found in the Codepen you linked with the version built in v1.3.4 and found that they [did not match](https://www.diffchecker.com/Ynfsel1j). Can you confirm that the version of drift you are using is correct? 

Here is a codepen I forked off of yours with the most up-to-date `Drift.min.js` included: https://codepen.io/sherwinski/pen/GbbGYP

**Comment by billythekid** [comment]
Hey @sherwinski,

I’ll confirm this tomorrow when back at a computer but at the moment I can already confirm that the issue exists in the forked pen.

Will double check my drift version in the morning but it’s latest version from npm so probably highest release but as I say, will confirm or not tomorrow.

Thanks

**Comment by sherwinski** [comment]
> This bug I’m finding is a tap bug but the zoom behaves differently to when handletouch is true. The bubble can’t be dragged around, just subsequent taps move it.

Just so that I'm clear: you're hoping to trigger the zoom on tap + hold (but not tap) and that the bubble can be dragged around? Or you would like no zoom to occur in either case?
Maybe @frederickfogerty can offer some insights here.

**Comment by billythekid** [comment]
Yeah I would think there should be no zoom at all when handleTouch is set to false.

I was just pointing out how it's not like it's ignoring handleTouch because the behaviour is different from this issue. When handleTouch is enabled you can smoothly drag the zoom around which sounds like that was the issue in #220. In this particular case the behaviour is different as it only appears on tap and can't be dragged.

To answer the question about version I'm using `"drift-zoom": "^1.3.4"` in my package.json file and this code in my js file to attach Drift objects to the images on the images I want them on:
```js
import Drift from 'drift-zoom';

window.initDriftZoomAll = function () {
  var images = document.querySelectorAll('.product__images__image');
  for (var i = 0; i < images.length; i++) {
    initDriftZoom(images[i]);
  }
};

window.DriftProductImages = [];

function initDriftZoom(image) {
  if (window.DriftProductImages.indexOf(image) === -1) {
    window.DriftProductImages.push(image);
    new Drift(image, {
      inlinePane: true,
      containInline: true,
      zoomFactor: 1.75,
      hoverDelay: 500,
      handleTouch: false
    });
  }
}

if (document.querySelector('.product__images__image') !== null) {
  window.initDriftZoomAll();
}
```

I'm using webpack (laravel mix) to compile.

**Comment by sherwinski** [comment]
Hey @billythekid,

Thanks for confirming the version. After doing some more digging I think I see the issue now. Even if we set `handleTouch` to false, tapping the image on mobile will still trigger a `mouseenter` event, which in turn will activate the zoom.
```js
  _bindEvents() {
    this.settings.el.addEventListener("mouseenter", this._handleEntry, false);
    this.settings.el.addEventListener("mouseleave", this._hide, false);
    this.settings.el.addEventListener("mousemove", this._handleMovement, false);


    if (this.settings.handleTouch) {
      this.settings.el.addEventListener("touchstart", this._handleEntry, false);
      this.settings.el.addEventListener("touchend", this._hide, false);
      this.settings.el.addEventListener("touchmove", this._handleMovement, false);
    }
  }
```
So as you pointed out it, is a different issue from #220. It won't matter if `handleTouch` is being set correctly if we're ultimately still triggering `_handleEntry()`. This might also explain the differences in drag behavior that you mentioned. 

It may take some time for me to get a solution out for this, but a PR would be welcome if you'd like to take a stab at it. And if you are looking for a quick-fix for this in the meantime, I would suggest [disabling drift](https://github.com/imgix/drift#driftdisable) conditional on whether you are on mobile vs desktop.

**Comment by billythekid** [comment]
Thanks for this,

Yeah I know that [detecting touchscreens](https://stackoverflow.com/a/4819886) is [notoriously difficult](http://www.stucox.com/blog/you-cant-detect-a-touchscreen/) to do accurately but in my case I can probably disable drift on touchstart or actually just media query `display: none;` in css at the smaller screen layout as a filthy workaround for now.

Thanks for looking into this and getting back to me.

**Comment by sherwinski** [comment]
Hey @billythekid, this fix is released now as part of [v1.3.5](https://github.com/imgix/drift/releases/tag/v1.3.5) 🎉
